src/Microsoft.AspNetCore.Mvc.RazorPages/Filters/PageHandlerExecutedContext.cs

```diff
@@ -112,5 +114,10 @@ public virtual ExceptionDispatchInfo ExceptionDispatchInfo
         /// Gets or sets the <see cref="IActionResult"/>.
         /// </summary>
         public virtual IActionResult Result { get; set; }
+
+        /// <summary>
+        /// Gets the <see cref="ViewDataDictionary"/>.
+        /// </summary>
+        public ViewDataDictionary ViewData { get; }
```

Comment:
Unlike controllers which initialize a vanilla Vdd, pages produce a Vdd<T> where T is the model type. This is cached deep in the belly of the beast. It's harder to have a PageVddFactory without relayering some of the caching code. This seemed like a smaller albeit more useful change.